crates/bevy_render/src/camera/camera.rs

```diff
@@ -61,6 +62,67 @@ impl Camera {
         let screen_space_coords = (ndc_space_coords.truncate() + Vec2::ONE) / 2.0 * window_size;
         Some(screen_space_coords)
     }
+
+    /// Given a position in screen space, compute the world-space line that corresponds to it.
+    pub fn screen_to_world_line(
+        pos_screen: Vec2,
+        windows: &Res<Windows>,
```

Comment:
This should probably be `&Windows` instead of `&Res<Windows>` so that in can be used e.g. in exclusive world access (same for the other uses of `Res<_>`)

Reply:
Done (using #2189)